README.md

```diff
@@ -100,335 +143,398 @@ immunopepper make_bg \
+- `--n-samples-lim-cancer`: Default = None. This parameter corresponds to the number of samples threshold in cohort specific filtering. It indicated the minimum number of cancer samples in which one should see an expression higher than `--cohort-expr-support-cancer` in order to consider the kmer as a cancer candidate. Kmers with an expression higher than `--cohort-expr-support-cancer` in at least `--n-samples-lim-cancer` samples will be considered as cancer candidates. The samples provided under `--ids_cancer_samples` will not be included in cohort filtering.
+- `--path-cancer-matrix-segm`: Default = None. Path to the cancer matrix containing segment expression from samples belonging to a cohort. The matrix will have the following dimensions: [kmers * samples]. However, it is more interesting to find novel junction kmers, so a junction expression matrix provided under `--path-cancer-matrix-edge` is the best expression proxy for that kmers. It is advised to provide only `--path-cancer-matrix-edge` and skip the inclusion of this file. If both matrices are provided, junction expression will be chosen in case there is expression information for the same kmer in both matrices. This will be the output 5 of [output, mode build](#output-files)
+- `--path-cancer-matrix-edge`: Default = None. Path to the cancer matrix containing junction expression from samples belonging to a cohort. The matrix will have the following dimensions: [kmers * samples]. This is the best expression proxy for junction kmers. It is advised to provide only this file and skip the inclusion of `--path-cancer-matrix-segm`. If both matrices are provided, junction expression will be chosen in case there is expression information for the same kmer in both matrices. This will be the ouput 6 of [output, mode build](#output-files).
+- `--cancer-support-union`: Default = False. Parameter to choose how the sample specific filtering and the cohort specific filtering are combined. By default, they are combined by choosing the common kmers to both filtering steps, i.e. performing an intersection. If this parameter is set to True, the union of both filtering steps will be performed, i.e. the kmers that pass either the sample specific filtering or the cohort specific filtering will be kept.
+
+*Optional parameters for the addition of additional backgrounds*: Parameters to add additional backgrounds that will be removed.
+- `--path-normal-kmer-list`: Default = None. List of kmers to be added as part of the normal samples. The kmers provided in this list will be included in the normal background, without having to pass any of the filtering steps for the normal data. Format: It can be either a *tsv* or *parquet* file. If *parquet* is the used format, the file should contain kmer in the first column.
```

Comment:
-> It can be either a *tsv* or *parquet* file. If *parquet* is the used format, the file should contain the header "kmer" in the first column.